src/it/resources/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/InputVerticalWhitespace.java

```diff
@@ -92,11 +92,11 @@ public int compareTo(InputVerticalWhitespace obj) { // ok
       return 0;
     }
     Class2 anon = // violation ''VARIABLE_DEF' should be separated from previous line.'
-            new Class2() {
-              public int compareTo(InputVerticalWhitespace obj) { // ok
-                int number = 0;
-                return 0;
-              }
-            };
+        new Class2() {
+          public int compareTo(InputVerticalWhitespace obj) { // ok
+            int number = 0;
+            return 0;
+          }
+        };
```

Comment:
formatter changed indentation. Not sure why the indentation wasn't changed during "formatting inputs" issue

Reply:
this is exactly a reason why we do this effort now to let CI check formatting for each commit.